lib/core/src/lib/datatable/components/datatable/datatable.component.html

```diff
@@ -28,15 +28,15 @@
             </div>
 
             <!-- Columns -->
-            <div *ngIf="multiselect" class="adf-datatable-cell-header adf-datatable-checkbox">
+            <th *ngIf="multiselect" class="adf-datatable-cell-header adf-datatable-checkbox" role="columnheader">
```

Comment:
There was a PR doing exactly the opposite recently, and we removed th/tr from here... so I suggest checking the history and contacting the corresponding devs